### ISSUE TITLE
How much energy is in a RF unit?

### ISSUE THREAD

**Issue by DrPlantabyte** [TeamBR-Modding/NeoTech#34]
Hi TeamBR,

I'm working on improving the interface between Power Advantage, Neotech, Open Computers, and Progressive Automation. However, there seems to be a mismatch between these mods in how much RF comes from a piece of coal. Neotech furnaces consume 1 RF per tick and smelt at the same speed as a vanilla furnace (200 ticks), meaning that 1 piece of coal is worth 1600 RF (nice and simple). However, the lowest tier RF generator in Progressive Automation generates 20 RF per tick, equating to a minimum of 32,000 RF per piece of coal. The Progressive Automation machines appear to generally use 40 RF per tick, making a single machine in Progressive Automation eat power like 40 NeoTech furnaces.

Thus, if I balance my power conversion ratios around NeoTech, then Progressive Automation machines require absurd amounts of energy, but if I balance the conversion ratio around Progressive Automation, NeoTech (and Open Computers) will essentially cost no energy. Power Advantage treats all RF machines equally, so I can't pick and choose.

Put simply, how much RF should burning a piece of coal in Power Advantage provide to all RF mods?

-DrCyano

**Comment by DrPlantabyte** [comment]
Actually, I think it would be better to just use accept Power Advantage energy directly, taking electricity at a ratio of 1 RF per 32 units of electricity (and 1 RF per 1 unit of steam). I'll try to learn enough scala to send you a pull request in the coming days.
